app/scripts/directives/multiSelectMenu.js

```diff
@@ -58,17 +58,17 @@ angular.module('gsUiInfraApp')
                 }
 
                 /**
-                 * Bind data to ng-model
+                 * call onchange function when model change
```

Comment:
It is a fix, since the onchange event was never trigger, even when the model changed.
I assume as it was binding to an attribute on the scope that no longer exists - it was an old implementation code that was not changed.

We are now changing the model using ngModelController api , so I changed it to check the model itself directly.